contracts/KyberOasisReserve.sol

```diff
@@ -0,0 +1,258 @@
+pragma solidity 0.4.18;
+
+
+import "./ERC20Interface.sol";
+import "./Utils.sol";
+import "./Withdrawable.sol";
+import "./KyberReserveInterface.sol";
+
+
+contract OtcInterface {
```

Comment:
what is otc?
please document
also reference to contract would be nice...

Reply:
as described this is Oaseis direct naming convention. Actually not sure why they use it.